HealthBarGump LT-Highlight

Alias-Alan opened the pull request:
ClassicUO - LastTarget Highlight (Red Border)
![Syrupz-UO-LastTargetHighlight](https://user-images.githubusercontent.com/53979689/76257711-979bb780-6228-11ea-9dc8-89137e9da600.gif)

Was a small feature request by Merrit. Sounded like a proper request so I dusted off the cobwebs. This simply highlights the custom bar with a RED border(similar to tabbing into war-mode for your personal health bar). Once a mobile is set to last target, their health bar (if opened, in range, and not your own) will now have a red border around it to make identifying last target selected easier for players.

Alias-Alan committed:
HealthBarGump+

ClassicUO - LastTarget Highlight (Red Border)
https://i.imgur.com/GSgo6QD.mp4

andreakarasho commented on src/Game/UI/Gumps/HealthBarGump.cs:
```diff
@@ -530,7 +530,27 @@ public override void Update(double totalMS, double frameMS)
 
+                if (TargetManager.LastTarget != World.Player && !_outOfRange)
+                {
+                    if (mobile == TargetManager.LastTarget)
```
mobile could be null.
Use LocalSerial

andreakarasho commented on src/Game/UI/Gumps/HealthBarGump.cs:
```diff
@@ -530,7 +530,27 @@ public override void Update(double totalMS, double frameMS)
+                            _border[1].LineColor = _border[2].LineColor = _border[3].LineColor = HPB_COLOR_RED;
+                        }
+                    }
+                    else if (mobile != TargetManager.LastTarget)
```
mobile could be null.
Use LocalSerial

Alias-Alan replied:
fixed both problems by throwing null check first before checking lasttarget

Alias-Alan committed:
safeguard +redundancy 

check if mobile is null before checking is lasttarget to work if mobile == null

Alias-Alan committed:
Update HealthBarGump.cs

Alias-Alan reviewed (commented):
I added the check though I feel it's rather redundant no? The entity null check is done before declaring mobile so it should never even get to the section I added is it returns null. Though perhaps I'm missing a way for the mobile to be destroyed after it's declaration at that point?
